src/OSPSuite.Presentation/Presenters/Importer/ImporterDataPresenter.cs

```diff
@@ -189,9 +182,9 @@ private void setDefaultMetaData()
 
       private void createSheetsForViewing()
       {
-         foreach (var sheet in _dataSourceFile.DataSheets.KeyValues)
+         foreach (var sheet in _dataSourceFile.DataSheets)
          {
-            _sheetsForViewing[sheet.Key] = sheet.Value.RawData.AsDataTable();
+            _sheetsForViewing[sheet.SheetName] = sheet.AsDataTable();
```

Comment:
WE started to rename those methods from AsDataTable to ToDataTable. Let's do it here as well

Reply:
done